dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/Fixture.kt

```diff
@@ -1,17 +1,12 @@
 package pl.droidsonroids.testing.mockwebserver
 
-internal class Fixture {
-
-    var statusCode = 0
-        internal set
-    var body: String? = null
-        internal set
-    var headers: List<String> = emptyList()
-        internal set
-    var connectionFailure: Boolean = false
-        internal set
-    var timeoutFailure: Boolean = false
-        internal set
+internal data class Fixture(
```

Comment:
Is change in this file intended?
It looks unrelated.

Reply:
Yes, as from snakeyaml 1.30, fields are not found, so I had to convert them to remove the `internal set`.
Checking it now, the data class is not required, si I've reverted the changes, but I kept the `internal set` removal, otherwise snakeyaml does cannot write to the properties.

<img width="1546" alt="Screenshot 2025-04-23 at 16 52 51" src="https://github.com/user-attachments/assets/11c5e351-9a2f-4954-b32b-51522c42f4d1" />